webapp/src/app/core/modules/openapi/api/user.service.ts

```
@@ -216,4 +218,136 @@ export class UserService implements UserServiceInterface {
+     */
+    public updateUserSettings(userSettings: UserSettings, observe?: 'body', reportProgress?: boolean, options?: {httpHeaderAccept?: 'application/json', context?: HttpContext, transferCache?: boolean}): Observable<UserSettings>;
+    public updateUserSettings(userSettings: UserSettings, observe?: 'response', reportProgress?: boolean, options?: {httpHeaderAccept?: 'application/json', context?: HttpContext, transferCache?: boolean}): Observable<HttpResponse<UserSettings>>;
+    public updateUserSettings(userSettings: UserSettings, observe?: 'events', reportProgress?: boolean, options?: {httpHeaderAccept?: 'application/json', context?: HttpContext, transferCache?: boolean}): Observable<HttpEvent<UserSettings>>;
+    public updateUserSettings(userSettings: UserSettings, observe: any = 'body', reportProgress: boolean = false, options?: {httpHeaderAccept?: 'application/json', context?: HttpContext, transferCache?: boolean}): Observable<any> {
+        if (userSettings === null || userSettings === undefined) {
+            throw new Error('Required parameter userSettings was null or undefined when calling updateUserSettings.');
+        }
+
+        let localVarHeaders = this.defaultHeaders;
+
+        let localVarHttpHeaderAcceptSelected: string | undefined = options && options.httpHeaderAccept;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)